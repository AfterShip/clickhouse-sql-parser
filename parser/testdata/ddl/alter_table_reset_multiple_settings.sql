ALTER TABLE example_table RESET SETTING max_part_loading_threads, max_parts_in_total, another_setting;