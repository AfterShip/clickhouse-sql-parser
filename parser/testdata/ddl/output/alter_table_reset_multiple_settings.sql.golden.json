[
  {
    "AlterPos": 0,
    "StatementEnd": 101,
    "TableIdentifier": {
      "Database": null,
      "Table": {
        "Name": "example_table",
        "QuoteType": 1,
        "NamePos": 12,
        "NameEnd": 25
      }
    },
    "OnCluster": null,
    "AlterExprs": [
      {
        "ResetPos": 26,
        "StatementEnd": 101,
        "Settings": [
          {
            "Name": "max_part_loading_threads",
            "QuoteType": 1,
            "NamePos": 40,
            "NameEnd": 64
          },
          {
            "Name": "max_parts_in_total",
            "QuoteType": 1,
            "NamePos": 66,
            "NameEnd": 84
          },
          {
            "Name": "another_setting",
            "QuoteType": 1,
            "NamePos": 86,
            "NameEnd": 101
          }
        ]
      }
    ]
  }
]